portfolio/src/main/webapp/my-travels/travels.js

```diff
@@ -0,0 +1,40 @@
+let placesCoords = {
```

Comment:
try to use `const` for immutable data.

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
Done.